nameserver/server_test.go

```diff
@@ -27,7 +27,7 @@ func TestDNSServer(t *testing.T) {
 	dnsAddr := fmt.Sprintf("localhost:%d", port)
 	testCIDR1 := testAddr1 + "/24"
 
-	common.InitDefaultLogging(true)
+	InitDefaultLogging(true)
```

Comment:
I have removed the `common` prefix, so I can use `Debug.Printf(...)` instead of the verbose `common.Debug.Printf(...)`